packages/domain-search/src/components/search-bar/index.tsx

```diff
@@ -39,7 +40,7 @@ export const SearchBar = () => {
 				onBlur={ () => {} }
 				onKeyDown={ () => {} }
 			/>
-			<DomainSearchControls.FilterButton count={ 3 } onClick={ () => {} } />
+			<Filter showTldFilter />
```

Comment:
It seems to me we don't want to show the filters at all if there are no available TLDs. In this case, instead of passing a prop, we can conditionally render the filter component.

Reply:
I think there will always be at least one available TLD, otherwise it wouldn't be possible to query for domain suggestions. We hide the filter in some specific flows, like the 100-year plan and domain ones.